docs/src/main/asciidoc/vault.adoc

```diff
@@ -497,6 +497,27 @@ You should see:
 {password=connor}
 ----
 
+== Vault Health Check
+
+If you are using the `quarkus-smallrye-health` extension, `quarkus-vault` can add a readiness health check
+to validate the connection to the Vault server. This is disabled by default.
+
+If enabled, when you access the `/health/ready` endpoint of your application you will have information about the connection validation status.
+
+This behavior can be enabled by setting the `quarkus.vault.health.enabled` property to `true` in your `application.properties`.
+
+Only if Vault is initialized, unsealed and active, the health endpoint returns that Vault is ready to serve requests.
+
+You can change a bit this behaviour by using `quarkus.vault.health.standbyok` and `quarkus.vault.health.perfstandbyok` to `true` in your `application.properties`.
+
+standbyok:: Specifies if being a standby should still return the active status code instead of the standby status code.
```

Comment:
```suggestion
stand-by-ok:: Specifies if being a standby should still return the active status code instead of the standby status code.
```
would be more readable maybe?

Reply:
I understand your point and I thought the same but in Vault, the property is named in this way `standbyok` so  I decided to leave it in the same name as in Vault so anyone familiar with Vault properties finds the same property. Do you think it has sense or better to adopt to the Quarkus way?